README.md

```diff
@@ -250,6 +250,16 @@ the [Lombok Gradle Plugin](https://plugins.gradle.org/plugin/io.freefair.lombok)
 to delombok your source code before it is passed to the Checker Framework
 for typechecking. This plugin does not support any other use of Lombok.
 
+## Compatibility with other annotation processors
+
+The plugin runs checkers by creating a custom manifest file containing the 
+checkers you specify, and then allowing `javac`'s processor auto-discovery 
+feature to find them. If you already provide
+a `-processor` option to `javac`, you **cannot** use this plugin: you
```

Comment:
This run-on sentence is confusing.  Does the text after the colon refer to your inability to use this plugin, or to how to work around the problem?
Another solution would be for the programmer to use auto-discovery.  Is there a reason that is not mentioned as an option?
Does the plugin respect existing manifest files and augment them?  (I determined the answer is "yes" by reading the code.  But removing the discussion of manifest files from the user manual means that the user manual doesn't have to even discuss this.)